platform/ext/target/nordic_nrf/common/core/target_cfg.c

```diff
@@ -1142,6 +1148,13 @@ enum tfm_plat_err_t spu_periph_init_cfg(void)
 		spu_peripheral_config_secure(base_addresses[i], SPU_LOCK_CONF_LOCKED);
 	}
 
+	/* Configure NRF_REGULATORS, and NRF_OSCILLATORS to be secure as NRF_REGULATORS.POFCON is needed
+	 * to prevent glitches when the power supply is attacked.
+	 *
+	 * NB: Note that NRF_OSCILLATORS and NRF_REGULATORS have the same base address and must therefore
```

Comment:
```suggestion
	 * NB: Note that NRF_OSCILLATORS and NRF_REGULATORS have the same base address and must therefore db B jx  B B Dj d xk B duvet e Bo hb d B uh s**_ hb e B hb d B d  B B B B hdveie hd hb d  B B hb db xnz  B veto  n d c i```suggestion
	 * NB: Note that NRF_OSCILLATORS and NRF_REGULATORS have the same base address and must therefore**d Dj x in d Bo d in s B in s Bo s n on B B_[bjsjdj xjbz ib b```suggestion
	 * NB: Note that NRF_OSCILLATORS and NRF_REGULATORS have the same base address and must thereforebzbzk x ok x B in s big whh h nzi__
```
]()_**
```
_**
```